### PR TITLE
docs: fix format issue

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/main/PULL_REQUESTS.md)

Commit Message:

Fix format issue to pass pre-push check.

Additional Description:
Risk Level: Low
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: docs/root/version_history/current.rst [hunk 1]
```diff
@@ -60,15 +60,14 @@ Removed Config or Runtime
...
-
-* crash support: restore crash context when continuing to processing requests or responses as a result of an asynchronous callback that invokes a filter directly. This is unlike the call stacks that go through the various network layers, to eventually reach the filter. For a concrete example see: ``Envoy::Extensions::HttpFilters::Cache::CacheFilter::getHeaders`` which posts a callback on the dispatcher that will invoke the filter directly.
-* http: added support for :ref:`original IP detection extensions<envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.original_ip_detection_extensions>`.
   Two initial extensions were added, the :ref:`custom header <envoy_v3_api_msg_extensions.http.original_ip_detection.custom_header.v3.CustomHeaderConfig>` extension and the
   :ref:`xff <envoy_v3_api_msg_extensions.http.original_ip_detection.xff.v3.XffConfig>` extension.
+* http: added support for :ref:`original IP detection extensions<envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.original_ip_detection_extensions>`.
```
Comment (PiotrSikora) [inline docs/root/version_history/current.rst:70]: This looks wrong. You moved the first line (`http: added support ...`) after the second line (`Two initial extensions were added ...`) from the same entry.

Comment (daixiang0) [reply docs/root/version_history/current.rst:70]: Oh I miss something.

**Review by PiotrSikora — APPROVED** [review]
Thanks!

**Comment by PiotrSikora** [comment]
This is not a flake, retesting it over and over again won't help:
```
Code coverage for source/common/quic is lower than limit of 88.4 (87.8)
```
Broken in #16522.

**Comment by phlax** [comment]
> This is not a flake

as we now have 2 permafails on `main` someone with override CI power needs to land a fix to one or other first

**Pull request merged**